templates/cli/scoop/appwrite.json.twig

```diff
@@ -0,0 +1,18 @@
+{
+    "$schema": "https://raw.githubusercontent.com/ScoopInstaller/Scoop/master/schema.json",
+    "version": "{{ sdk.version }}",
+    "description": "Appwrite cli project manager",
+    "license": "BSD-3-Clause",
+    "url": "https://github.com/{{ sdk.gitUserName }}/{{ sdk.gitRepoName | caseDash }}/releases/download/{{ sdk.version }}/{{ language.params.executableName }}-cli-win-x64.exe",
+    "bin": [
+        [
+            "{{ language.params.executableName }}-cli-win-x64.exe",
+            "{{ language.params.executableName|caseLower }}"
+        ]
+    ],

```

Comment:
This should be changed to include ARM now that Scoop supports it ScoopInstaller/Scoop#3146.